pyproject.toml

```
@@ -83,6 +86,11 @@ show_error_codes = true
 warn_return_any = true
 warn_unused_ignores = true
 
+# Prevent mypy from trigger the missing import error for our written packages
+[[tool.mypy.overrides]]
+module = ["discord_bot.*", "src.discord_bot.*"]
+ignore_missing_imports = true
+
 [build-system]
 requires = ["poetry-core"]
-build-backend = "poetry.core.masonry.api"
+build-backend = "poetry.core.masonry.api"
```

need extra line at the end

```suggestion
build-backend = "poetry.core.masonry.api"

```